pkg/ccl/importccl/import_stmt_test.go

```diff
@@ -6069,11 +6072,11 @@ table_name FROM [SHOW TABLES] ORDER BY (schema_name, table_name)`,
 		// In the case of the test, the ID of the 3 schemas that will be cleaned up
```

Comment:
nit: update comment to mention public schema descriptor, and edit the increments below to be `publicSchemaID + 1` and so on.

Reply:
Done.